docs/fundamentals/documentarray/embedding.md

```diff
@@ -1,130 +1,157 @@
 (embed-via-model)=
 # Embed via Neural Network
 
-```{important}
+When DocumentArray `.tensors` being set,
+you can use a neural network to {meth}`~docarray.array.mixins.embed.EmbedMixin.embed` it into it's vector representations,
+i.e. filling `.embeddings`.
```

Comment:
```suggestion
You can set a DocumentArray's `tensors` attribute by using a neural network to {meth}`~docarray.array.mixins.embed.EmbedMixin.embed` vectors.
```

Can you change the text for `{meth}`~docarray.array.mixins.embed.EmbedMixin.embed` vectors.` to `create vector embeddings`? I need to brush up on my RST syntax.